### PR TITLE
Add Flysystem achievement

### PR DESCRIPTION
**Pull request by tgalopin**

### CODE REVIEW

File: achievements/flysystem.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Flysystem / Symfony integration achievements
+
+> By Titouan Galopin (@tgalopin)
+
+In January 2018, I stumbled a problem with autowiring: how can one easily and
```
Comment (greg0ire) [inline achievements/flysystem.md]:
```suggestion
In January 2018, I stumbled upon a problem with autowiring: how can one easily and 
```

File: achievements/flysystem.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Flysystem / Symfony integration achievements
+
+> By Titouan Galopin (@tgalopin)
+
+In January 2018, I stumbled a problem with autowiring: how can one easily and 
+efficiently autowire different implementations of the same interface? In my 
+specific case, the problem was linked to the Flysystem library, in which having 
+multiple different filesystems can happen often.
+
+This problem lead to a vast conversation throughout 2018 with Nicolas and
```
Comment (greg0ire) [inline achievements/flysystem.md]: We know who you are talking about, but I'm not sure everyone will :)
```suggestion
This problem lead to a vast conversation throughout 2018 with Nicolas Grekas and 
```

File: achievements/flysystem.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+This problem lead to a vast conversation throughout 2018 with Nicolas and 
+ultimately was the reason to introduce an idea of "semantics" using the param 
+annotation ([#27172](https://github.com/symfony/symfony/pull/27172)).
+
+However, this implementation had many problems and was not well suited to 
+solve the problem. It was later closed when the introduction of named aliases 
+solved the problem much more elegantly 
+([#28234](https://github.com/symfony/symfony/pull/28234)). This feature was released in 
+Symfony 4.2, in november 2018, and the hackathon was even the occasion for 
+Gregoire Paris to write its documentation
```
Comment (greg0ire) [inline achievements/flysystem.md]: :blush:

File: achievements/flysystem.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+could be achieved. This is one of the thing I worked on during this hackathon: 
+creating this integration, as an official Symfony bundle for Flysystem.
+
+The repository for this project is available at 
+[https://github.com/thephpleague/flysystem-bundle](https://github.com/thephpleague/flysystem-bundle). 
+It's not entierely finished but it will be in the coming days.
+
+The aim of this integration is to become the Symfony official filesystem 
+abstraction, by providing a great documentation, leveraging all the powerful 
+features of Flysystem and creating a Symfony Flex recipe.
```
Comment (OskarStark) [inline achievements/flysystem.md]: Missing new line

File: achievements/flysystem.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+However, this implementation had many problems and was not well suited to 
+solve the problem. It was later closed when the introduction of named aliases 
+solved the problem much more elegantly 
+([#28234](https://github.com/symfony/symfony/pull/28234)). This feature was released in 
+Symfony 4.2, in november 2018, and the hackathon was even the occasion for 
+Gregoire Paris to write its documentation 
+([#11339](https://github.com/symfony/symfony-docs/pull/11339)).
+
+Now that named aliases are available, a much better integration with Flysystem 
+could be achieved. This is one of the thing I worked on during this hackathon:
```
Comment (greg0ire) [inline achievements/flysystem.md]:
```suggestion
could be achieved. This is one of the things I worked on during this hackathon: 
```

File: achievements/flysystem.md (hunk 3 above)
Comment (OskarStark) [reply achievements/flysystem.md]: Grégoire avec accent aigu? 🇫🇷

Comment (greg0ire) [reply achievements/flysystem.md]: Yes please :pray: 
```suggestion
Grégoire Paris to write its documentation 
```

**Comment by tgalopin** [comment]
Updated!

**Review by greg0ire — APPROVED** [review]

**Review by OskarStark — APPROVED** [review]

File: achievements/flysystem.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
+# Flysystem / Symfony integration achievements
```
Comment (jfhovinne) [inline achievements/flysystem.md]: I would prefer a '-' instead of a '/'.

File: achievements/flysystem.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+In January 2018, I stumbled upon a problem with autowiring: how can one easily and 
+efficiently autowire different implementations of the same interface? In my 
+specific case, the problem was linked to the Flysystem library, in which having 
+multiple different filesystems can happen often.
+
+This problem lead to a vast conversation throughout 2018 with Nicolas Grekas and 
+ultimately was the reason to introduce an idea of "semantics" using the param 
+annotation ([#27172](https://github.com/symfony/symfony/pull/27172)).
+
+However, this implementation had many problems and was not well suited to
```
Comment (jfhovinne) [inline achievements/flysystem.md]: 'many issues' maybe (to avoid repeating the word) ?

File: achievements/flysystem.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+
+This problem lead to a vast conversation throughout 2018 with Nicolas Grekas and 
+ultimately was the reason to introduce an idea of "semantics" using the param 
+annotation ([#27172](https://github.com/symfony/symfony/pull/27172)).
+
+However, this implementation had many problems and was not well suited to 
+solve the problem. It was later closed when the introduction of named aliases 
+solved the problem much more elegantly 
+([#28234](https://github.com/symfony/symfony/pull/28234)). This feature was released in 
+Symfony 4.2, in november 2018, and the hackathon was even the occasion for
```
Comment (jfhovinne) [inline achievements/flysystem.md]: November (N uppercase)

File: achievements/flysystem.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+Grégoire Paris to write its documentation 
+([#11339](https://github.com/symfony/symfony-docs/pull/11339)).
+
+Now that named aliases are available, a much better integration with Flysystem 
+could be achieved. This is one of the things I worked on during this hackathon: 
+creating this integration, as an official Symfony bundle for Flysystem.
+
+The repository for this project is available at 
+[https://github.com/thephpleague/flysystem-bundle](https://github.com/thephpleague/flysystem-bundle). 
+It's not entierely finished but it will be in the coming days.
```
Comment (jfhovinne) [inline achievements/flysystem.md]: entirely

**Review by jfhovinne — CHANGES_REQUESTED** [review]
Thanks! Minor fixes needed.

**Comment by tgalopin** [comment]
Updated

**Review by jfhovinne — APPROVED** [review]

**Comment by jfhovinne** [comment]
Thanks!

**Pull request merged**